modules/seqexec/engine/src/test/scala/seqexec/engine/SequenceSpec.scala

```diff
@@ -54,8 +56,10 @@ class SequenceSpec extends AnyFlatSpec {
 
   }
 
+  private implicit def logger: Logger[IO] = Slf4jLogger.unsafeFromName[IO]("seqexec-engine")
```

Comment:
This is deprecated (here and elsewhere) in favor of `getLoggerFromName` … if you're not getting a deprecation warning you may need to upgrade.